src/Components/Aspire.Kafka.Consumer/IConsumerConnection.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Confluent.Kafka;
+
+namespace Aspire.Kafka.Consumer;
+
+/// <summary>
+/// A connection to a Kafka broker to consume messages.
+/// </summary>
+/// <typeparam name="TKey">The type of the key of a message.</typeparam>
+/// <typeparam name="TValue">The type of the value of a message.</typeparam>
+public interface IConsumerConnection<TKey, TValue>
```

Comment:
We're trying really hard to avoid abstractions owned by the aspire components.

Reply:
I see. I'm trying to avoid collisions with types not owned by aspire that may have been registered in DI by user/customer code. I could have made these new types internal but then unit testing would became painful as once built `IProducer<TKey, TValue>` and `IConsumer<TKey, TValue>` does not expose their configuration values.